README.md

```diff
@@ -1 +1 @@
-# lol
+# 'sup?
```

Comment:
I dunno, I think this should be more polite. How about "Hello, fellow code wranglers"?